content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -148,8 +189,19 @@ spec:
   # to this address.
   - address: <IPofCluster2IngressGateway>
     ports:
-      http1: 15443 # Do not change this port value
-      tcp2: 15443 # Do not change this port value
+      http1: 15443
+      tcp2: 15443
+{{< /text >}}
+
+__Note__: If your Kubernetes cluster is running in an environment that does not
+support external load-balancers, use the following commands to obtain the correct
+`endpoints.address` and `http1`/`tcp2` port values of the remote `istio-ingressgateway`
+service:
+
+{{< text bash >}}
+$ NODE=$(kubectl get ep/istio-ingressgateway -n istio-system -o=jsonpath='{.subsets[0].addresses[0].nodeName}')
+$ kubectl get nodes/$NODE -o=jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}'
+$ kubectl get svc -n istio-system istio-ingressgateway -o=jsonpath='{.spec.ports[?(@.name=="tls")].nodePort}'
```

Comment:
Maybe this would clearer if mention that you need to use the nodePort for port 15433 and then make the command:
```
kubectl get svc -n istio-system istio-ingressgateway -o=jsonpath='{.spec.ports[?(@.port==15443)].nodePort}'
```